client/client_role_test.go

```diff
@@ -18,9 +18,12 @@ import (
 	"context"
 	"math/rand"
 	"sync"
+	"testing"
 	"time"
 
+	"github.com/stretchr/testify/assert"
```

Comment:
I could probably write a CI script which checks the grouping automatically :laughing:

Reply:
It's not really that important I think.